doit/dependency.py

```diff
@@ -625,11 +640,16 @@ def get_status(self, task, tasks_dict):
             try:
                 file_stat = os.stat(dep)
             except OSError:
+                if rebuild_log is not None:
+                    rebuild_log.append('Dependent file "{}" does not exist.'.format(dep))
                 raise DependencyException("Dependent file '{}' does not exist."
```

Comment:
Since rebuild_log will be returned we need to find another solution for this case instead of raising an exception.